app/src/main/java/org/mozilla/focus/telemetry/SentryWrapper.kt

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.telemetry
+
+import android.content.Context
+import io.sentry.Sentry
+import io.sentry.android.AndroidSentryClientFactory
+import org.mozilla.focus.BuildConfig
+
+/**
+ * An interface to the Sentry crash reporting SDK. All code that touches the Sentry APIs
+ * directly should go in here (like TelemetryWrapper).
+ *
+ * With the current implementation, to enable Sentry on Beta/Release builds, add a
+ * <project-dir>/.sentry_dsn_release file with your key. To enable Sentry on Debug
+ * builds, add a .sentry_dsn_debug key and replace the [DataUploadPreference.isEnabled]
+ * value with true (upload is disabled by default in dev builds).
+ */
+object SentryWrapper {
+
+    fun init(context: Context) {
+        onIsEnabledChanged(context, DataUploadPreference.isEnabled(context))
```

Comment:
What if this is changed later, this doesn't get re-inited right? I guess it's not a big deal if we miss a Sentry upload if someone decides to turn it on after the app starts, and this is simpler.

Reply:
We always call `Sentry.init` when the activity is created, even if it's disabled. We pass in an invalid client if it's disabled, which prevents upload (I need to verify this doesn't record events while it's disabled either).

Whenever the toggle changes, we call init with a new client, which should resume sentry again.

We don't need to call our own init more than once because it all ends up in the same place.